src/DynamoCore/Models/RunSettings.cs

```diff
@@ -22,9 +22,9 @@ public class RunSettings : NotificationObject
         private bool runTypesEnabled;
 
         /// <summary>
-        /// Boolean indicates if Dynamo should block run even when mode is Automatic.
+        /// This static property with true value will be used when opening a dyn file with RunMode = Automatic and we don't want to execute the graph
```

Comment:
Let's update the comment, this is not only used for Automatic but blocking any run mode

Reply:
@QilongTang comment updated, check the next commit: https://github.com/DynamoDS/Dynamo/pull/12954/commits/6756fafc79131f36453908e5484beb3b9c5b2437